code/missionui/missionscreencommon.cpp

```diff
@@ -313,27 +313,40 @@ void set_active_ui(UI_WINDOW *ui_window)
 	Active_ui_window = ui_window;
 }
 
-void common_music_init(int score_index)
+SCP_string common_music_get_filename(int score_index)
 {
-	if ( Cmdline_freespace_no_music ) {
-		return;
+	if (Cmdline_freespace_no_music) {
+		return SCP_string();
 	}
 
-	if ( score_index >= NUM_SCORES ) {
+	if (score_index >= NUM_SCORES) {
 		Int3();
```

Comment:
I'd use this opportunity to get rid of the Int3() here.

Reply:
Done, replaced with Assertion.